doc/proposals/TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-## <Title of Proposal>
+# <Title of Proposal> + Proposal for Operator SDK
```

Comment:
I don't really think we should use the "Proposal for Operator SDK" suffix for the proposal titles. In fact, IIRC the proposals are the ones that were turning up in searches ahead of the actual documentation, so I'm not sure we want to make the proposals _more_ searchable.